ledger/participant-integration-api/src/main/scala/platform/store/backend/UpdateToDbDto.scala

```diff
@@ -173,11 +174,15 @@ object UpdateToDbDto {
         )
 
       case u: TransactionAccepted =>
-        incrementCounterForEvent(
-          metrics.daml.indexerEvents,
-          IndexedUpdatesMetrics.Labels.eventType.transaction,
-          IndexedUpdatesMetrics.Labels.status.accepted,
-        )
+        withOptionalMetricLabels(
+          IndexedUpdatesMetrics.Labels.applicationId -> u.optCompletionInfo.map(_.applicationId)
+        ) { implicit mc: MetricsContext =>
+          incrementCounterForEvent(
+            metrics.daml.indexerEvents,
+            IndexedUpdatesMetrics.Labels.eventType.transaction,
+            IndexedUpdatesMetrics.Labels.status.rejected,
```

Comment:
I don't think the change of the status value is expected.
```suggestion
            IndexedUpdatesMetrics.Labels.status.accepted,
```